ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -253,6 +254,23 @@ ipv6_access_lists:
         action: "< action as string >"
 ```
 
+#### MAC Access-Lists
```

Comment:
Since this is a new model, we should move to list of dicts now. We will convert all the existing ones in 4.0.
So I suggest:
```yaml
mac_access_lists:
  - name: < mac_access_list_name_1 >
    counters_per_entry: < true | false >
    entries:
      - sequence: < sequence_id_1 >
        action: "< action as string >"
```